package.json

```diff
@@ -1,26 +1,36 @@
 {
   "name": "@ents24/be-sure",
+  "version": "1.1.2",
   "description": "NPM module for terse, 'throwy' validation",
+  "type": "module",
   "publishConfig": {
     "access": "public"
   },
-  "version": "1.1.2",
-  "main": "index.js",
-  "devDependencies": {
-    "jest": "^29.3.1"
-  },
   "scripts": {
-    "test": "jest"
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "prepublishOnly": "sh -c 'if [ -f ./haltPublish.js ]; then node ./haltPublish.js; fi'",
```

Comment:
This could be inverted, i.e. confirm that we are in the `dist` directory rather than just not in the root directory.

For example, I could run the publish process from within the `tests` dir. I think this is just a question of how paranoid we want to be with this sanity check.

Reply:
You can only publish from a package root i.e. a directory containing a `package.json`, so your run-from-test-dir example is bogus.

But I do agree that "only run in /dist" is a better narrative than "do not run if this rando JS script is present". I'll make that change if it's quick